pandas/core/generic.py

```diff
@@ -1924,11 +1924,17 @@ def _repr_latex_(self):
     If you wish to write to more than one sheet in the workbook, it is
     necessary to specify an ExcelWriter object:
 
-    >>> writer = pd.ExcelWriter('output2.xlsx', engine='xlsxwriter')
-    >>> df1.to_excel(writer, sheet_name='Sheet1')
-    >>> df2 = df1.copy()
-    >>> df2.to_excel(writer, sheet_name='Sheet2')
-    >>> writer.save()
+    >>> with pd.ExcelWriter('output.xlsx') as writer:
+    ...     df1.to_excel(writer, sheet_name='Sheet_name_1')
+    ...     df2 = df1.copy()
```

Comment:
Can you do this copy outside of the context manager? On initial glance I assumed it to be some requirement with the context manager itself, though that's clearly not the case